examples/spawn-and-move/manifests/base/base.toml

```diff
@@ -1,3 +1,2 @@
 kind = "Class"
-class_hash = "0x679177a2cb757694ac4f326d01052ff0963eac0bc2a17116a2b87badcdf6f76"
```

Comment:
Hum, @lambda-0x do you have an idea why the class hash here is not taken anymore? Is this expected?

Reply:
compiler always write class hash during compilation, maybe it was mistakenly removed from the file manually? @remybar can you retry running build?